Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,22 @@
 import React from 'react';
 import './Header.css';
+import HeaderTitle from './HeaderTitle';
+import HeaderContent from './HeaderContent';
+import ImageThumbnail from './ImageThumbnail';
+
+
+const HeaderContainer = () => {
+  return (
+    <div className = 'header-container'>
+      <ImageThumbnail/>
+      <div className = 'headerText'>
```

Comment:
You don't need a space between the className and the assignment. This should be written as `className='headerText'`